lib/assert.js

```diff
@@ -572,7 +559,6 @@ function _throws(shouldThrow, block, expected, message) {
 
 // Expected to throw an error.
 // assert.throws(block, Error_opt, message_opt);
-
 assert.throws = function throws(block, /*optional*/error, /*optional*/message) {
```

Comment:
Also could you remove all the `/*optional*/` is driving my IDE (WebStorm) crazy.